### PR TITLE
Comment out erroneous error catch

### PR DESCRIPTION
**Pull request by tgperring**
Commented out a test that was incorrect in vals2distr.m

### CODE REVIEW

File: herbert_core/utilities/maths/vals2distr.m [hunk 1]
```diff
@@ -31,9 +31,9 @@
 flags = {'norm','poisson'};
 [par,keyval,~,~,ok,mess] = parse_arguments (varargin,keyval_def,flags);
 if ~ok, error(mess), end
-if numel(par)>1
-    error('Check number of input arguments')
-end
+% if numel(par)>1
```
Comment (nickbattam-tessella) [inline herbert_core/utilities/maths/vals2distr.m]: If this is erroneous why not delete it?

Comment (tgperring) [reply herbert_core/utilities/maths/vals2distr.m]: THis is really me just checking that I can check in, run Jenkins, so I am just commenting out and will edit properly later (there are other changes to be performed)

I.e. testing Toby understands what is going on!

Comment (tgperring) [reply herbert_core/utilities/maths/vals2distr.m]: ...and it seems that I am not a member of the project on Jenkins. I can log into Jenkins, but do not see results of this test.

**Comment by nickbattam-tessella** [comment]
Build failures are unrelated to this PR -- see https://github.com/pace-neutrons/Herbert/issues/113

**Comment by cmarooney-stfc** [comment]
@nickbattam-tessella @tgperring This has now been opened almost one year! And seems to have been approved but is marked do not merge. Could one of you update the state and maybe merge it?

**Comment by abuts** [comment]
I think this test PR is not needed any more

**Pull request closed without merging**